packages/core/src/shared/settings-amazonq.gen.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * This file was autogenerated by generateSettings.ts. Do NOT modify by hand.
+ * To update this file, update the settings in the extension package.json
+ * and run `npm run generateSettings` for that extension.
+ */
+
+export const amazonqSettings = {
+    "amazonQ.telemetry": {},
```

Comment:
nit: should we include these specific .gen.ts files in our linter/prettier?

Reply:
no, there would be some extra effort to achieve proper formatting on gen/commit (albeit quite small) but I don't see any value here because its not meant for humans